insights/parsers/satellite_postgresql_query.py

```diff
@@ -8,12 +8,16 @@
 -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 SatelliteComputeResources - command ``psql -d foreman -c 'select name, type from compute_resources' --csv``
 -----------------------------------------------------------------------------------------------------------
-SatelliteSCAStatus - command ``psql -d candlepin -c "select displayname, content_access_mode from cp_owner" --csv``
--------------------------------------------------------------------------------------------------------------------
-SatelliteKatelloEmptyURLRepositories - command ``psql -d foreman -c 'select id, name from katello_root_repositories where url is NULL;' --csv``
------------------------------------------------------------------------------------------------------------------------------------------------
 SatelliteCoreTaskReservedResourceCount - command ``psql -d pulpcore -c 'select count(*) from core_taskreservedresource' --csv``
 -------------------------------------------------------------------------------------------------------------------------------
+SatelliteKatelloEmptyURLRepositories - command ``psql -d foreman -c 'select id, name from katello_root_repositories where url is NULL;' --csv``
```

Comment:
If this parser is going to be deprecated, it would be better to remove it from the list.

Reply:
@xiangce updated, please help to review again, thanks.